### PR TITLE
Your garden page

### PR DESCRIPTION
**Pull request by lewy192**
Your garden page is completed, ready for review and then merge.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -125,7 +128,7 @@ a {
 
 @media screen and (min-width: 768px) {
     header {
-        padding: 5px 0 5px 0;
+        padding: 5px 8% 5px 8%;
```
Comment (ClareRadtke) [inline assets/css/style.css]: These can be simplified to padding: 5px 8% (top right bottom left ) where top/bottom and left/right are the same value you can list them once and it reads the same

File: assets/scripts/widgetScript.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ function updateWeatherWidgets() {
         const lat = userCoordinatesObject.usersLatitude;
         const excludeParams = "minutely";
         // unix here is temp, could embed this function in the url itself -- for testing i will leave it as a variable here
-        const yesterdayUnix = (moment().unix() - 86400 * 4) / 1;
+        const yesterdayUnix = (moment().unix() - 86400) / 1;
```
Comment (ClareRadtke) [inline assets/scripts/widgetScript.js]: Great work finding and resolving this issue

File: responseExample.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+```
```
Comment (ClareRadtke) [inline responseExample.md:1]: could delete this file if its no longer needed

File: yourGarden.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="/assets/css/reset.css">
+        <link rel="stylesheet" href="/assets/css/style.css">
+        <link rel="stylesheet" href="/assets/css/yourGardenStyles.css">
+        <script src="/assets/scripts/widgetScript.js" defer></script>
+        <script src="/assets/scripts/yourGardenScript.js" defer ></script>
```
Comment (ClareRadtke) [inline yourGarden.html]: remove the first / to correct these file paths and link them up.

File: yourGarden.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                        <a class ='nav-link'>Contact</a>
+                    </li>
+                    <li class = 'nav-links'> 
+                        <a class ='nav-link'>Gallery</a>
+                    </li>
+                </ul>
+            
+                <div class = 'widget-container'>
+                <div class = "weather-icon-container">
+                    <img class = 'weather-icon' src="/assets/images/widget-icons/load-bar.svg" alt = 'load bar icon'>
```
Comment (ClareRadtke) [inline yourGarden.html]: same as above remove first / to link up

File: index.html [hunk 6]
```diff
@@ -39,47 +40,47 @@
     </header>
     <section class="hero">
         <div class="container">
-            <div id="your-Garden"  class="your-garden content-card">
+            <a id="your-Garden"  class="your-garden content-card" href = "/yourGarden.html">
```
Comment (ClareRadtke) [inline index.html]: remove the first /

**Review by ClareRadtke — CHANGES_REQUESTED** [review]
In widgetScript.js line 77 const baseSrc = "/assets/images/widget-icons/"; remove the first / to link this up correctly - it is erroring

**Review by lewy192 — COMMENTED** [review]
all done

**Review by ClareRadtke — DISMISSED** [review]
Nice work

**Review by ClareRadtke — APPROVED** [review]
Great work resolving those conflicts!

**Pull request merged**